freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -66,11 +71,85 @@ def handle(self, event):
             "Orignial nvrs of build in the advisory #{0} are: {1}".format(
                 db_event.search_key, " ".join(original_nvrs)))
 
-        msg = "Skipping due to being blocked on further implementation for now."
+        index_images = self._pyxis.get_operator_indices()
+        # get latest bundle images per channel per index image filtered by higher version
+        all_bundles = self._pyxis.get_latest_bundles(index_images)
+        # get images by nvrs and then get their digests
+        rebuilt_images_digests = self.get_digests_by_nvrs(original_nvrs)
+        if not rebuilt_images_digests:
+            log.warning(f"The are no digests for nvrs: {','.join(original_nvrs)}")
+            return []
+
+        bundles = self._pyxis.filter_bundles_by_digests(rebuilt_images_digests,
+                                                        all_bundles)
+        bundle_digests = []
+        for bundle in bundles:
+            if not bundle.get('bundle_path_digest'):
+                log.warning(f"Bundle {bundle['bundle_path']} doesn't have"
+                            " 'bundle_path_digests' set")
+                continue
+            bundle_digests.append(bundle['bundle_path_digest'])
+        bundle_images = self._pyxis.get_images_by_digests(bundle_digests)
+
+        # now we have nvrs of bundles to rebuild
+        bundles_nvrs = self._filter_bundle_images_by_builds(  # noqa: F841
+            bundle_images, rebuilt_images_digests)
 
         # Skip that event because we can't proceed with processing it.
         # TODO
-        # Next step would be queries Pyxis and getting the digests of these nvrs
+        # Now when we have bundle images' nvrs we can procceed with rebuilding it
+        msg = "Skipping due to being blocked on further implementation for now."
```

Comment:
It'd be nice to know how many Freshmaker would have rebuilt.
```suggestion
        msg = f"Skipping the rebuild of {len(bundle_nvrs)} bundle images due to being blocked on further implementation for now."
```